src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConfiguration.java

```diff
@@ -1,17 +1,32 @@
 package uk.gov.companieshouse.chipsrestinterfacesconsumer.configuration;
 
+import org.apache.avro.Schema;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.producer.KafkaRestClient;
 import uk.gov.companieshouse.kafka.consumer.CHKafkaConsumerGroup;
 import uk.gov.companieshouse.kafka.consumer.ConsumerConfig;
 import uk.gov.companieshouse.kafka.deserialization.DeserializerFactory;
+import uk.gov.companieshouse.kafka.producer.Acks;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.kafka.producer.ProducerConfig;
+import uk.gov.companieshouse.kafka.producer.ProducerConfigHelper;
 
 import java.util.Collections;
 
 @Configuration
 class KafkaConfiguration {
 
+    @Value("${SCHEMA_REGISTRY_URL}")
+    private String schemaRegistryUrl;
+
+    @Value("${CHIPS_SCHEMA_URI}")
```

Comment:
I'd be more specific here and name it CHIPS_REST_INTERFACES_SCHEMA_URI as chips_schema could also imply chips filing consumer schema.

Reply:
Renamed